### PR TITLE
chore: gh workflow preamble

### PR DESCRIPTION
**Pull request by frezbo**
Add preamble to the generated GitHub workflow yaml.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
```
Comment (frezbo) [inline .github/workflows/ci.yaml:1]: this doesn't seems to get updated when I generate with a newer version of kres, not sure why? :thinking:

Comment (smira) [reply .github/workflows/ci.yaml:1]: it should be updated only if the file has changes without the header

File: internal/project/golang/generate.go [hunk 2]
```diff
@@ -348,8 +348,7 @@ func (generate *Generate) CompileTemplates(output *template.Output) error {
 
 	output.Define(fmt.Sprintf("%s/version.go", generate.versionPackagePath()), templates.VersionGo).
 		PreamblePrefix("// ").
-		WithLicense().
-		NoOverwrite()
```
[the remark this answers is not in the document]
Comment (smira) [reply internal/project/golang/generate.go:352]: I think it needs to be generated once, and then `make generate` should update it

[the remark this answers is not in the document]
Comment (smira) [reply internal/project/golang/generate.go:352]: I think we should still generate once, and if needed, one can delete existing version.go if it needs to be updated.

[the remark this answers is not in the document]
Comment (frezbo) [reply internal/project/golang/generate.go:352]: makes sense, updating.

**Review by smira — APPROVED** [review]

**Pull request merged**